### PR TITLE
Fix rewards FE errors

### PR DESCRIPTION
**Pull request by valentinludu**
1st error: Rewards applicants returned a null value.
2nd error: Mapped Fragment without keys

### CODE REVIEW

File: components/common/DatabaseEditor/components/properties/UserAndRoleSelect.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ function SelectedOptions({
     <>
       {value.map((option) => {
         return (
-          <>
+          <Fragment key={option.id}>
```
Comment (mattcasey) [inline components/common/DatabaseEditor/components/properties/UserAndRoleSelect.tsx:92]: thanks for catching

File: components/common/DatabaseEditor/components/propertyValueElement.tsx [hunk 2]
```diff
@@ -482,7 +482,7 @@ function PropertyValueElement(props: Props) {
     Array.isArray(propertyValue) &&
     !card.fields.isAssigned
   ) {
-    propertyValueElement = null;
+    propertyValueElement = <Box />;
```
Comment (mattcasey) [inline components/common/DatabaseEditor/components/propertyValueElement.tsx:485]: this doesn't seem right, null is a valid value to pass in leiu of a JSX element

**Review by mattcasey — CHANGES_REQUESTED** [review]
see question

File: components/common/DatabaseEditor/components/propertyValueElement.tsx (hunk 2 above)
Comment (valentinludu) [reply components/common/DatabaseEditor/components/propertyValueElement.tsx:485]: At line 705 that `null` set at line 485 is logging an error.
I could verify that it's not a `REWARDS_APPLICANTS_BLOCK_ID` before showing the error.

**Review by mattcasey — APPROVED** [review]

**Pull request merged**